README.md

```diff
@@ -73,6 +73,33 @@ after_success:
   - Rscript -e 'covr::codecov()'
 ```
 
+If you are using [Appveyor CI](http://ci.appveyor.com), and are not using
```

Comment:
This should just be something like 

```
on_success:
  - Rscript -e "covr::codecov()"
```
e.g. https://github.com/r-lib/fs/blob/3bb86eaf969b06f439970b982cdc849646332e46/appveyor.yml#L29-L30

Reply:
Simplified accordingly. Please go ahead if you would like to reword anything else in the ReadMe.